packages/sdks/snippets/react/src/routes/targeted-page/is-logged-in-request.tsx

```diff
@@ -0,0 +1,15 @@
+import { fetchOneEntry, GetContentOptions } from '@builder.io/sdk-react';
+
+export default function targetedRequest(
+  urlPath: string,
+  { apiKey, model }: GetContentOptions
+) {
+  return fetchOneEntry({
+    apiKey,
+    model,
+    userAttributes: {
+      isLoggedIn: true,
+      urlPath,
+    },
+  });
+}
```

Comment:
The arguments of `targetedRequest` add a layer of indirection that can complicate the code snippet for the end-user.

I would stick with a philosophy of "spread everything" to keep this wrapper function as unobtrusive as possible:

```suggestion
import { fetchOneEntry, GetContentOptions } from '@builder.io/sdk-react';

export default function targetedRequest(options: GetContentOptions) {
  return fetchOneEntry({
    ...options,
    userAttributes: {
      isLoggedIn: true,
      ...options.userAttributes,
    },
  });
}
```

Reply:
@samijaber Is your assumption then that the arguments provided to `targetedRequest()` will look something like this?

```javascript
const options = {
  apiKey: BUILDER_API_KEY,
  model: MODEL,
  userAttributes: {
    urlPath: window.location.pathname,
  },
};
```

As someone new to the code, I feel like having `urlPath` explicit within the code snippet feels important as the request will not work without it. Do you think something like this is too verbose?

```javascript
import { fetchOneEntry, GetContentOptions } from '@builder.io/sdk-react';

export default function targetedRequest(options: GetContentOptions) {
  return fetchOneEntry({
    ...options,
    userAttributes: {
      isLoggedIn: true,
      urlPath: options.userAttributes?.urlPath,
      ...options.userAttributes, // Include additional userAttributes
    },
  });
}
```